specification/maps/data-plane/Microsoft.Maps/Data/preview/2.0/data.json

```diff
@@ -250,7 +250,7 @@
             }
           },
           "401": {
-            "description": "Access denied due to invalid subscription key or invalid Azure Active Directory (Azure AD) bearer token.  Make sure to provide a valid key for an active Azure subscription and Maps resource.  Otherwise, verify the [WWW-Authenticate](https://tools.ietf.org/html/rfc6750#section-3.1) header for error code and description of the provided Azure AD bearer token.",
+            "description": "Access denied due to invalid subscription key or invalid  Microsoft Entra ID bearer token.  Make sure to provide a valid key for an active Azure subscription and Maps resource.  Otherwise, verify the [WWW-Authenticate](https://tools.ietf.org/html/rfc6750#section-3.1) header for error code and description of the provided Microsoft Entra ID bearer token.",
```

Comment:
There's an extra space in front of "Microsoft Entra ID", same for the following lines: 318, 407, 475, 534

Reply:
fixed